app/Models/FeedDAOMySQL.php

```diff
@@ -0,0 +1,33 @@
+<?php
+declare(strict_types=1);
+
+class FreshRSS_FeedDAOMySQL extends FreshRSS_FeedDAO {
+
+	#[\Override]
+	public function updateCachedValues(int ...$feedIds): int|false {
+		// Performance problem when using `WHERE id_feed IN (...)`, so cannot take advantage of `$feedIds` for now with MySQL / MariaDB
+		$sql = <<<'SQL'
+			UPDATE `_feed`
+			JOIN (
+				SELECT id_feed,
+					COUNT(id) AS total_entries,
+					SUM(CASE WHEN is_read = 0 THEN 1 ELSE 0 END) AS unread_entries
+				FROM  `_entry`
+				-- WHERE id_feed IN (...)
```

Comment:
We could make an option for that, but really I would prefer something that works well in every MariaDB / MySQL setup...